Tests/YamsTests/EncoderTests.swift

```diff
@@ -94,8 +102,9 @@ import Yams
                 XCTFail("Failed to encode \(T.self) to YAML.")
             }
 
-            if let expectedYAML = yaml {
-                XCTAssertEqual(expectedYAML, payload, "Produced YAML not identical to expected YAML.")
+            if let expectedYAML = yamlString {
+                let producedYAML = String(data: payload, encoding: .utf8)! // swiftlint:disable:this force_unwrapping
```

Comment:
Yup. But I prefer failing output as
```
XCTAssertEqual failed: ("{}
") is not equal to ("{ }
") - Produced YAML not identical to expected YAML.
```
to
```
XCTAssertEqual failed: ("Optional("{}\n")") is not equal to ("Optional("{ }\n")") - Produced YAML not identical to expected YAML.
```
Especially in long strings, I feel the former is easier to see.